supabase/migrations/20260718000904_eve_audit_tracer.sql

```
@@ -0,0 +1,80 @@
+-- Eve audit tracer bullet (issue #419).
+--
+-- Audit rows are app-owned, service-role-only, append-only records. Their
+-- schema intentionally has no raw prompt, model transcript, request payload,
+-- payment, or PII columns. Redaction is applied by the server before insert.
+
+CREATE INDEX eve_run_summaries_updated_idx
+    ON public.eve_run_summaries (updated_at DESC);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)